test/middleware/AuthMiddleware.test.ts

```diff
@@ -0,0 +1,127 @@
+import 'reflect-metadata';
+import '../../src/controllers/index';
+import { expect, assert } from 'chai';
+import * as request from 'supertest';
+import { PENALTY_DETAILS_PAGE_URI, OTHER_REASON_DISCLAIMER_PAGE_URI, OTHER_REASON_PAGE_URI } from '../../src/utils/Paths';
+import { createApp, getDefaultConfig } from '../ApplicationFactory';
+import { Maybe, VerifiedSession } from 'ch-node-session-handler';
+import { createFakeSession } from '../utils/session/FakeSessionFactory';
+import { SessionKey } from 'ch-node-session-handler/lib/session/keys/SessionKey';
+import { generateSessionId, generateSignature } from 'ch-node-session-handler/lib/utils/CookieUtils';
+
+const config = getDefaultConfig();
+const id = generateSessionId();
+const sig = generateSignature(id, config.cookieSecret);
+
+const session = createFakeSession(
+    [{
+        [SessionKey.Id]: id
+    }, {
+        [SessionKey.ClientSig]: sig
+    }], config.cookieSecret, true);
+
+const authedApp = createApp(session);
```

Comment:
Since it is only used in first describe block I suggest you move it inside the block